Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CalculatedCameraExtrinsics.cs

```diff
@@ -22,5 +24,28 @@ public override string ToString()
         {
             return $"Succeeded: {Succeeded} {base.ToString()}";
         }
+
+        public byte[] Serialize()
+        {
+            var str = JsonUtility.ToJson(this);
+            var payload = Encoding.ASCII.GetBytes(str);
```

Comment:
To avoid surprises if content ends up containing Unicode characters, it might be safer to use Encoding.UTF8 for all of these string transfers. UTF8 is the default encoding that BinaryWriter and BinaryReader use for string encoding.